internal/auth/auth.go

```diff
@@ -30,7 +30,7 @@ var requiredScopes = []string{
 	"create:clients", "delete:clients", "read:clients", "update:clients",
 	"create:resource_servers", "delete:resource_servers", "read:resource_servers", "update:resource_servers",
 	"create:rules", "delete:rules", "read:rules", "update:rules",
-	"read:client_keys", "read:logs",
+	"read:client_keys", "read:logs", "read:users", "update:users",
```

Comment:
We don't need delete?

Reply:
not yet, deleting user blocks only require `update:users`